### PR TITLE
Solution of 세균 증식, 내적, 두 개 뽑아서 더하기, 시저 암호, N개의 최소공배수, 최솟값 만들기

### PR DESCRIPTION
**Pull request by sjh-judeee3**

### CODE REVIEW

File: level-1/내적.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+def solution(a, b):
+    answer = 0
+    for i in range(len(a)):
+        answer += a[i] * b[i]
```
Comment (juyoungml) [inline level-1/내적.py]: Try to convert this code using Python's list comprehension. It should be a good exercise.

File: level-1/두-개-뽑아서-더하기.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+def solution(numbers):
+    answer = []
+    for i in range(len(numbers)):
+        for j in range(i+1, len(numbers)):
+            if numbers[i]+numbers[j] not in answer:
+                answer.append(numbers[i] + numbers[j])
```
Comment (juyoungml) [inline level-1/두-개-뽑아서-더하기.py]: I think there's some room to improve time complexity. Analyze time complexity of current algorithm and try to find out with a better solution.

File: level-1/시저-암호.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+def solution(s, n):
+    answer = ''
+    s = list(s)
+    
+    for letter in s: 
+        #아스키코드 변환
+        a = ord(letter)
+        
+        if letter.isupper(): #대문자일 경우
+            a = ((a - 65) + n) % 26 + 65
```
Comment (juyoungml) [inline level-1/시저-암호.py]: 65 and 97 doesn't seem to be interpretable. Can you replace those constants?

File: level-2/N개의-최소공배수.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+def gcd(a, b):
+    if a > b:
+        a, b = b, a
+    while b%a != 0:
+        b, a = a, b%a
```
Comment (juyoungml) [inline level-2/N개의-최소공배수.py]: Some comments about this part would be great :)

File: level-2/N개의-최소공배수.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    return a
+
+def solution(arr):
+    answer = 1
+    arr = sorted(arr)
```
Comment (juyoungml) [inline level-2/N개의-최소공배수.py]: Think about the space complexity of this algoithm. If you haven't heard about it, I think it is a good time to understand this concept. 

Do you know the difference between `sort` and `sorted`? Which one is better in terms of space complexity and why? When should we use the one with the more space complexity?

File: level-2/N개의-최소공배수.py [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+
+    #두 개의 수끼리 최소공배수를 구하고 최소공배수와 다음 수의 최소공배수 구하기
+    for i in range(1, len(arr)):
+        gcd_num = gcd(arr[i-1], arr[i])
+        answer = int(arr[i-1] * arr[i] / gcd_num)
```
Comment (juyoungml) [inline level-2/N개의-최소공배수.py]: Can you avoid using `int`?

**Review by juyoungml — CHANGES_REQUESTED** [review]
Please check my comments.

File: level-1/두-개-뽑아서-더하기.py (hunk 2 above)
Comment (sjh-judeee3) [reply level-1/두-개-뽑아서-더하기.py]: Current algorithm has time complexity of O(n^3).

File: level-2/N개의-최소공배수.py (hunk 4 above)
Comment (sjh-judeee3) [reply level-2/N개의-최소공배수.py]: Comment on that part is added.

File: level-1/내적.py (hunk 1 above)
Comment (sjh-judeee3) [reply level-1/내적.py]: Completed fixing code with list comprehension.

File: level-1/시저-암호.py (hunk 3 above)
Comment (sjh-judeee3) [reply level-1/시저-암호.py]: Changed 65 into `ord('A')` and 97 into `ord('a')`.

**Review by juyoungml — COMMENTED** [review]
LGTM! How about other ones?

File: level-2/N개의-최소공배수.py (hunk 6 above)
Comment (sjh-judeee3) [reply level-2/N개의-최소공배수.py]: Line 15 has been changed into `answer = arr[i-1] * arr[i] // gdc_num`.

**Comment by sjh-judeee3** [comment]
최솟값-만들기.py proof: https://sjh-judeee3.tistory.com/3
Space complexity: https://sjh-judeee3.tistory.com/2

I need some help with finding space complexities of `sort` and `sorted`.

File: level-1/두-개-뽑아서-더하기.py (hunk 2 above)
Comment (sjh-judeee3) [reply level-1/두-개-뽑아서-더하기.py]: The new algorithm has the time complexity of O(n^2).

File: level-2/N개의-최소공배수.py (hunk 5 above)
Comment (juyoungml) [reply level-2/N개의-최소공배수.py]: Nevermind. This requires too much information to answer... You don't have to answer it. Just keep in mind that `sorted` returns a new array.

**Review by juyoungml — APPROVED** [review]
LGTM!

**Pull request merged**